### PR TITLE
docs(GettingStarted): Updates for FRMW-22

### PR DESCRIPTION
**Pull request by parlarjb**
Miscellaneous changes for JIRA task [FRMW-22](https://jira.rax.io/browse/FRMW-22)

### CODE REVIEW

File: guides/using-encoreui.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
-# Using Encore Framework in your Project
+# Using EncoreUI In Your Own Project
 
 ## Installation
 
 ### Easy Setup: Clone the EncoreUI-Template
 
-[EncoreUI-Template](https://github.com/rackerlabs/encore-ui-template) is a project template to help get things started with new EncoreUI based apps. You must be a Rackspace employee in order to access the repository.
+The [Encore Generator](https://github.com/rackerlabs/generator-encore) is a project template to help get things started with new EncoreUI-based apps. You must be a Rackspace employee in order to access the repository. Instructions for using it are documented at the project page.
```
Comment (glynnis) [inline guides/using-encoreui.md]: "at the project page" is a little unclear. Maybe a link/better wording?

Comment (parlarjb) [inline guides/using-encoreui.md]: The link is right at the beginning. I was actually thinking about removing the "Instructions for using..." completely, as it's not really necessary. If someone clicks on the Encore Generator link, they'll see the instructions right away.

Comment (glynnis) [inline guides/using-encoreui.md]: :+1: for removing that sentence.

**Comment by parlarjb** [comment]
I forgot that we have grunt tasks that assume `getting-started.md` exists, I need to update those

File: guides/using-encoreui.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
-# Using Encore Framework in your Project
+# Using EncoreUI In Your Own Project
 
 ## Installation
 
 ### Easy Setup: Clone the EncoreUI-Template
```
Comment (julianbonilla) [inline guides/using-encoreui.md]: This title needs an update too.

File: README.md [hunk 3]
```diff
@@ -4,9 +4,9 @@
 
 EncoreUI is a library of reusable AngularJS widgets for internal Rackspace projects. While a few components are specific to Rackspace, most are generic enough to be used by any app.
 
-# Getting Started
+# Using EncoreUI In Your Own Project
 
-[A 'Getting Started' guide](./guides/getting-started.md) is available to help new projects get off the ground quickly.
+We have a guide for [Using EncoreUI In Your Own Project](./guides/using-encoreui.md) to help new projects get off the ground quickly.
```
Comment (TheNando) [inline README.md]: Would it be more consistent to not capitalize this link? In all the other links, only proper nouns are capitalized.

**Comment by TheNando** [comment]
Only one suggestion, then LGTM.

File: README.md (hunk 3 above)
Comment (CITguy) [inline README.md]: Maybe: 


> To help guide you in getting a new project off the ground quickly, please reference ["Using EncoreUI In Your Own Project"](./guides/using-encoreui.md).

**Comment by parlarjb** [comment]
I took care of @TheNando's comment, using @CITguy's suggestion

**Comment by CITguy** [comment]
LGTM

**Pull request merged**